scripts/atlantis/docker-build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
```

Comment:
Maybe take `scripts/docker-build.sh` and tweak it so it takes `DOCKER_REPO` and `DIR` as parameters?

Reply:
I did have a PR for something like that : https://github.com/accurics/terrascan/pull/555

I'll re-open it and make these changes there later. savvy?